### PR TITLE
feat: add styles and docs for ul elements

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

File: packages/components/src/components.d.ts
Comment (gfellerph) [inline packages/components/src/components.d.ts:1]: Can you remove this file from the PR? It's being updated in a dedicated PR.

File: packages/components/src/components/post-popovercontainer/readme.md
Comment (gfellerph) [inline packages/components/src/components/post-popovercontainer/readme.md:1]: Same here, does not belong in this one.

File: packages/documentation/src/stories/foundation/list/list-reset.sample.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+ul.my-list {
+  @include post.list-reset();
+}
```
Comment (gfellerph) [inline packages/documentation/src/stories/foundation/list/list-reset.sample.scss:5]: Including the whole styles library is not necessary for using the reset and the mixin name is `reset-list`.

```suggestion
@use '@swisspost/design-system-styles/mixins/utilities' as post;

ul.my-list {
  @include post.reset-list();
}
```

File: packages/documentation/src/stories/foundation/list/list.docs.mdx [hunk 2]
```diff
@@ -24,6 +25,16 @@ This only applies to immediate child list items, meaning you will need to add th
 
 <Canvas of={ListStories.UnstyledList} sourceState="shown" />
 
+#### Sass mixin
+
+If you wan't to use the class above, you can still reset the list styles with the following mixin:
```
Comment (gfellerph) [inline packages/documentation/src/stories/foundation/list/list.docs.mdx:30]:
```suggestion
If you want to use the class above, you can still reset the list styles with the following mixin:
```

File: packages/styles/src/elements/bullet-list.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    #{tokens.get('list-bullet-item-icon-size')} + #{tokens.get('list-bullet-item-icon-gap-inline')}
+  );
+
+  > li {
+    &::marker {
+      line-height: 1;
+    }
+  }
+
+  &.bullet {
```
Comment (gfellerph) [inline packages/styles/src/elements/bullet-list.scss:27]: Can we prefix the class name with the component name: `.list-bullet`? Could imagine naming conflicts with this class name.

(Would obviously have to be updated throughout the PR).

```suggestion
  &.list-bullet {
```

File: packages/styles/src/elements/_index.scss [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 @use 'body';
 @use 'anchor';
+@use 'bullet-list';
```
Comment (gfellerph) [inline packages/styles/src/elements/_index.scss:5]: Could be named `list-bullet` as well just to place it with all the other list types.

```suggestion
@use 'list-bullet';
```

**Review by gfellerph — CHANGES_REQUESTED** [review]
Solid implementation!

File: packages/documentation/src/stories/foundation/list/list-reset.sample.scss (hunk 1 above)
Comment (oliverschuerch) [reply packages/documentation/src/stories/foundation/list/list-reset.sample.scss:5]: Fixed the mixin name.

The core file only contains `variables`, `mixins` and `functions`. This means, that nothing is added to the CSS output, except what is actually used. With this in mind, it does not make sense, to add more specific import statements imo.
What do you think?

**Review by gfellerph — APPROVED** [review]

**Pull request merged**